### PR TITLE
Rename posted to boards and cleanup icons

### PR DESCRIPTION
**Pull request by defnax**
* Rename posted links to boards
* Cleanup removed unused old posted icons

its needs a make clean fresh compile i removed icons.

Why rename? the posted service is not more links only now it supports more 

**New in 2020:**
* Image sharing can be used as imageboard (like reddit/9GAG...)
* html posts to use for Blog posts with or without inline pictures.

Added 7 years ago:
* Links post rs or http links 

the discussion is on rs forums for renaming idea

### CODE REVIEW

**Comment by felisucoibi** [comment]
Boards is short, ok for me. But Log ins not News!

File: retroshare-gui/src/gui/Posted/PostedDialog.cpp [hunk 1]
```diff
@@ -93,13 +93,13 @@ UserNotify *PostedDialog::createUserNotify(QObject *parent)
 
 QString PostedDialog::getHelpString() const
 {
-	QString hlp_str = tr("<h1><img width=\"32\" src=\":/icons/help_64.png\">&nbsp;&nbsp;Posted</h1>    \
-    <p>The posted service allows you to share internet links, that spread among Retroshare nodes like forums and \
+	QString hlp_str = tr("<h1><img width=\"32\" src=\":/icons/help_64.png\">&nbsp;&nbsp;Boards</h1>    \
+    <p>The poards service allows you to share internet links, that spread among Retroshare nodes like forums and \
```
Comment (sehraf) [inline retroshare-gui/src/gui/Posted/PostedDialog.cpp]: typo

Comment (defnax) [reply retroshare-gui/src/gui/Posted/PostedDialog.cpp]: done

**Comment by G10h4ck** [comment]
I don't bother about posted/board, is it just me that feels that boards might be quite confusing respect to forums?

**Comment by defnax** [comment]
> I don't bother about posted/board, is it just me that feels that boards might be quite confusing respect to forums?

no i think not realy, own new name boards is better, the word "posted" or "links" is not more the right name for posted service on gui side.
Boards often used on reddit like clones, or reddit clones has own fantasy name branding.

> Posts are organized by subject into user-created boards called "subreddits", which cover a variety of topics including news, science, movies, video games, music, books, fitness, food, and image-sharing.

File: retroshare-gui/src/gui/gxs/GxsGroupShareKey.cpp [hunk 2]
```diff
@@ -93,9 +93,9 @@ void GroupShareKey::setTyp()
         if (!rsPosted)
             return;
         
-        ui->headerFrame->setHeaderImage(QPixmap(":/images/posted_64.png"));    
-        ui->headerFrame->setHeaderText(tr("Share topic admin permissions"));
-        ui->sharekeyinfo_label->setText(tr("You can allow your friends to edit the topic. Select them in the list below. Note: it is not possible to revoke Posted admin permissions."));
+        ui->headerFrame->setHeaderImage(QPixmap(":/icons/png/posted.png"));
```
Comment (csoler) [inline retroshare-gui/src/gui/gxs/GxsGroupShareKey.cpp:96]: There is no admin permission for boards (there is on channels, but not in Boards). Could you hide that menu entry?

**Pull request merged**